Cargo.toml

```diff
@@ -40,6 +40,11 @@ walker = "^1.0.0"
 [target.x86_64-unknown-linux-gnu.dependencies.inotify]
 version = "^0.1"
 
+[target.x86_64-unknown-linux-gnu.dependencies.mio]
+git = "https://github.com/carllerche/mio.git"
```

Comment:
The next `mio` milestone has a ETA for 3rd December, but I don't know whether `EventedFd` will be in there. I've asked on IRC for details.